lib/endpoints/class-wp-json-posts-controller.php

```diff
@@ -293,8 +296,11 @@ public function update_item( $request ) {
 		}
 
 		if ( ! empty( $schema['properties']['sticky'] ) && isset( $request['sticky'] ) ) {
-			$sticky = isset( $request['sticky'] ) ? (bool) $request['sticky'] : false;
-			$this->handle_sticky_posts( $sticky, $post_id );
+			if ( ! empty( $request['sticky'] ) ) {
```

Comment:
@danielbachhuber does this mean if you don't pass `sticky` _at all_ in an update request, it will remove `sticky` from it? (this might have been the behaviour before too - which is incorrect).